.jscsrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Ok, so perhaps we should start with a PR to get the jquery preset to be more comprehensive.